### ISSUE TITLE
train error

### ISSUE THREAD

**Issue by cploveflower** [D-X-Y/SAN#21]
Hello@D-X-Y:
I am very sorry to disturb you, but when I run the bash scripts/300W/300W_Cluster.sh 0,1 GTB 3,it show that :
runtimeerror:found 0 files in subfolder of :./cache_data/cache/AFLW .
It's because of the datasets?That I just download the 300w,and I don't download the AFLW

**Comment by D-X-Y** [comment]
You can simply delete the 19-th line in scripts/300W/300W_Cluster.sh, which is "--style_eval_root ./cache_data/cache/AFLW".

**Comment by cploveflower** [comment]
if I just download the 300W datasets,I change the --style_eval_root ./cache_data/cache/AFLW \ into 
 ./cache_data/cache/300W\,  does it have any influence for the cluster images into different groups?

**Comment by cploveflower** [comment]
when  I run the "bash scripts/300W/300W_Cluster.sh 0,1 GTB 3",I found the accuracy is very low,and the loss is very high,like below:
[Train = 226][5000/7272] accuracy :24.9 loss:1.3795
and other result all like this,I don't know why,does the result is the same with you?

**Comment by D-X-Y** [comment]
The accuracy should be much more higher. You can see my log at : https://drive.google.com/open?id=1DtUg0LoZXNuDxeqRtcRxrA4tjrN9rpJa

**Comment by cploveflower** [comment]
I just do it as your github have said,and I don't have the "--style_eval_root ./cache_data/cache/AFLW" as
 you suggest because I don't download the AFLW,and when I run the bash scripts/300W/300W_Cluster.sh 0,1 GTB 3,the epoch is 300,and the learning rate don't change,is still equal to 0.01。and the input_torch_size is ([2,3,224,224]),I don't know why.

**Comment by D-X-Y** [comment]
Why you have this line "[Train = 226] 5000/7272"? I only train the model with 2 epoch and one epoch should have 228 iterations.
 I just re-run the codes, it shows:
```
epoch : [0/2] lr=0.01
 [Train=000] [[2019-04-11 06:56:59]] [  0/228] accuracy : 23.4, loss : 1.4037, input:torch.Size([64, 3, 224, 224]), output:torch.Size([64, 4])
 [Train=000] [[2019-04-11 06:57:28]] [ 50/228] accuracy : 83.2, loss : 0.4329, input:torch.Size([64, 3, 224, 224]), output:torch.Size([64, 4])
 [Train=000] [[2019-04-11 06:57:57]] [100/228] accuracy : 87.3, loss : 0.3357, input:torch.Size([64, 3, 224, 224]), output:torch.Size([64, 4])
```
I also update the cluser.py to check whether your dataset is correct, you can try to pull the lastest update and re-run the code.

**Comment by cploveflower** [comment]
I have solved it,I think it's because of the python version,I have changed the python version in script,and it's successful,Thank you!

**Comment by D-X-Y** [comment]
You are welcome~
